bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/api/CoordinateSystem.java

```diff
@@ -41,71 +43,69 @@
      * like the top-down and the bottom-up, it operates in two where the top
      * (middle) rail closed value is 0 and the bottom (primary) rail closed position
      * is also 0 and fully open for both is 65535
-     * 
+     *
      * The position element can take on multiple states depending on the family of
      * shade under control.
      *
-     * The ranges of position integer values are 
+     * The ranges of position integer values are
      *      shades: 0..65535
      *      vanes: 0..32767
-     * 
-     * Shade fully up: (top-down: open, bottom-up: closed) 
-     *      posKind: 1 {ZERO_IS_CLOSED} 
+     *
+     * Shade fully up: (top-down: open, bottom-up: closed)
+     *      posKind: 1
      *      position: 65535
      *
-     * Shade and vane fully down: (top-down: closed, bottom-up: open) 
-     *      posKind: 1 {ZERO_IS_CLOSED}
+     * Shade and vane fully down: (top-down: closed, bottom-up: open)
+     *      posKind: 1
      *      position1: 0
-     * 
+     *
      * ALTERNATE: Shade and vane fully down: (top-down: closed, bottom-up: open)
-     *      posKind: 3 {VANE_COORDS}
+     *      posKind: 3
      *      position: 0
```

Comment:
There is a small error (pre-dates this PR, but probably worth fixing) in these comments. When the shade (primary rail) is fully up, a "top-down" shade is closed and a "bottom-up" shade is open, so the descriptions on line 54 are backwards. Similarly, when the shade/primary rail is fully down, the top-down shade is open and the bottom-up shade is closed, so the descriptions on lines 58 and 62 are also backwards. These changes will make this set of comments consistent with the comments on lines 37-41 that were taken from the Hunter Douglas API docs.

Reply:
Ok.